spinalcordtoolbox/deepseg/nnunet.py

```diff
@@ -14,7 +12,7 @@
 from nnunetv2.inference.predict_from_raw_data import nnUNetPredictor   # noqa: E402
 
 
-def create_nnunet_from_plans(path_model):
+def create_nnunet_from_plans(path_model, device):
```

Comment:
I was going to suggest putting another `torch.device` annotation here, but `torch` is no longer imported so we can't. I see that there's [a standard pattern for this situation](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING), but maybe that's too heavy.